spec/spec_helper.rb

```diff
@@ -81,7 +81,80 @@
     # (e.g. via a command-line flag).
     config.default_formatter = 'doc'
   end
+
+  # Check for leaky test resources
+  config.after(:all) do
```

Comment:
We already spoke about this work before, and I really like it -- it's mega valuable, but I'm slightly concerned about the user experience for new contributors to the gem.

Getting a big stack trace just by running rspec, or even an error without a stack trace (for things our patch can't catch) can be a bit confusing, especially because this is custom to our setup.

So here's what I suggest we can do about it:

1. Make this behavior non-breaking by default. The current version already includes a `CI` flag to decide if to break or not, but I suggest making the default not breaking -- that way we inform, but not scare.

2. Add a note about what this is to the `DevelopmentGuide.md` AND mention in the error message that we provide some tips there.

3. Indent the output a bit, so it's clear what belongs where :)

Reply:
Added all suggested changes. Output looks like this now:
<img width="868" alt="Screen Shot 2021-02-19 at 12 28 31 PM" src="https://user-images.githubusercontent.com/583503/108540871-ba937000-72af-11eb-96c5-47bbeb3934a8.png">
... after many, many lines, a screenshot of the last section:
<img width="352" alt="Screen Shot 2021-02-19 at 12 28 38 PM" src="https://user-images.githubusercontent.com/583503/108540876-bcf5ca00-72af-11eb-8888-bc8e88c0549f.png">